### PR TITLE
VACMS-2618: Export config changes from devshop environment.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #2618 

## Testing done
Behat

## QA steps
Visual environment comparison test to be performed by @rachel-kauff

### CODE REVIEW

File: config/sync/node.type.news_story.yml [hunk 1]
```diff
@@ -9,70 +9,26 @@ dependencies:
 third_party_settings:
   menu_ui:
     available_menus:
-      - va-alexandria-health-care
```
Comment (swirtSJW) [inline config/sync/node.type.news_story.yml:12]: THese menu removals are due to outdated stuff.   This hunk should not be committed.

File: config/sync/node.type.story_listing.yml [hunk 2]
```diff
@@ -10,75 +10,31 @@ third_party_settings:
   menu_ui:
     available_menus:
       - outreach-and-events
-      - va-alexandria-health-care
```
Comment (swirtSJW) [inline config/sync/node.type.story_listing.yml:13]: THese menu removals are due to outdated stuff.   This hunk should not be committed.

File: config/sync/node.type.vamc_operating_status_and_alerts.yml [hunk 3]
```diff
@@ -9,65 +9,21 @@ dependencies:
 third_party_settings:
   menu_ui:
     available_menus:
-      - va-alexandria-health-care
```
Comment (swirtSJW) [inline config/sync/node.type.vamc_operating_status_and_alerts.yml:12]: THese menu removals are due to outdated stuff.   This hunk should not be committed.

File: tests/behat/drupal-spec-tool/content_model_bundles.feature [hunk 4]
```diff
@@ -63,7 +63,7 @@ Feature: Content model bundles
 | Step by step | step_by_step | Paragraph type | An ordered list (1, 2, 3, 4, N) of steps. |
 | Step-by-Step | step_by_step | Content type | An ordered list (1, 2, 3, 4, N) of steps with Call to Action buttons. |
 | Story | news_story | Content type | Community stories highlight the role of a VA facility, program, or healthcare system in a Veteran's journey. They may be a case study of a specific patient, a description of a new or successful program, or a community-interest story. |
-| Stories list | story_listing | Content type | A listing of stories. |
+| Story listing page | story_listing | Content type | A listing of stories. |
```
Comment (swirtSJW) [inline tests/behat/drupal-spec-tool/content_model_bundles.feature:66]: Double check with Rachel.  I think this naming change is outdated.  Stories list I believe is the most up do date value.

File: tests/behat/drupal-spec-tool/content_model_content_type_fields.feature [hunk 5]
```diff
@@ -179,22 +179,22 @@ Feature: Content model: Content Type fields
...
+| Content type | Story | Full text | field_full_story | Text (formatted, long) |  | 1 | Text area (multiple rows) |  |
 | Content type | Story | Image | field_media | Entity reference |  | 1 | Media library | Translatable |
 | Content type | Story | Image caption | field_image_caption | Text (plain, long) |  | 1 | Textarea (multiple rows) with counter |  |
-| Content type | Story | Intro text | field_intro_text | Text (plain, long) | Required | 1 | Textarea (multiple rows) with counter | Translatable |
+| Content type | Story | First sentence (lede) | field_intro_text | Text (plain, long) | Required | 1 | Textarea (multiple rows) with counter | Translatable |
 | Content type | Story | Meta tags | field_meta_tags | Meta tags |  | 1 | Advanced meta tags form | Translatable |
 | Content type | Story | Order | field_order | List (integer) |  | 1 | Select list | Translatable |
 | Content type | Story | Owner | field_administration | Entity reference | Required | 1 | Select list | Translatable |
-| Content type | Story | Story listing | field_listing | Entity reference | Required | 1 | Select list | Translatable |
```
Comment (swirtSJW) [inline tests/behat/drupal-spec-tool/content_model_content_type_fields.feature:191]: I think this is unneccessary

File: config/sync/node.type.story_listing.yml [hunk 6]
```diff
@@ -10,75 +10,31 @@ third_party_settings:
...
       - va-wilmington-health-care
     parent: 'outreach-and-events:'
   node_title_help_text:
-    title_help: '<a href="https://design.va.gov/content-style-guide/page-titles-and-section-titles" target="_blank">Use sentence case</a>. '
+    title_help: 'Add a page title with sentence case capitalization (<a href="https://design.va.gov/content-style-guide/page-titles-and-section-titles" target="_blank">See further guidelines</a>). '
   node_revision_delete:
     minimum_revisions_to_keep: 50
     minimum_age_to_delete: 0
     when_to_delete: 0
-name: 'Stories list'
```
Comment (swirtSJW) [inline config/sync/node.type.story_listing.yml:81]: Check with Rachel.   I think this change is outdated.

File: config/sync/field.field.node.event.field_link.yml [hunk 7]
```diff
@@ -11,7 +11,7 @@ id: node.event.field_link
 field_name: field_link
 entity_type: node
 bundle: event
-label: 'URL of an external page or registration link for this event'
```
Comment (swirtSJW) [inline config/sync/field.field.node.event.field_link.yml:14]: Validating this change with Rachel too, as it seems to be going from something helpful to something cryptic.

**Comment by ethanteague** [comment]
I'm closing this one, @swirtSJW @rachel-kauff

**Pull request closed without merging**